### ISSUE TITLE
Does not work if the component is a folder with an `index.vue`.

### ISSUE THREAD

**Issue by tatyshev** [fussinatto/vscode-vue-peek#1]
https://github.com/fussinatto/vscode-vue-peek/blob/23fd3c2747f9b2e9bbc0d8dd3a88f6a45f239475/src/PeekFileDefinitionProvider.ts#L30

Possibly needs to do something similar to:
```javascript
vscode.workspace.findFiles(`**/${fileName}`, `**/${fileName}/index.vue`, '**/node_modules');
```

If this is the right way, can i make PR?

**Comment by fuzinato** [comment]
You are right, although suggested code will not work since second argument is "exclude" pattern ( node modules are not considered when searching). But please make a PR if you find a solution. Thanks

**Comment by fuzinato** [comment]
Issue is fixed and released in 1.0.1
